lib/std/net.zig

```diff
@@ -1920,6 +1925,15 @@ pub const StreamServer = struct {
                 &mem.toBytes(@as(c_int, 1)),
             );
         }
+        // TODO: Investigate if Windows needs an analogue.
```

Comment:
Windows do not have "REUSEPORT", instead it has "REUSEADDR". 
There is a good writeup about the subtle differences in this SO post -> https://stackoverflow.com/questions/14388706/how-do-so-reuseaddr-and-so-reuseport-differ

Windows doc on REUSEADDR: https://learn.microsoft.com/en-us/windows/win32/winsock/using-so-reuseaddr-and-so-exclusiveaddruse